.github/workflows/pr-check.yaml

```
@@ -103,11 +194,12 @@ jobs:
 
       # Install npm packages for quadlet extension
       - name: Execute pnpm in Podman Quadlet Extension
-        run: pnpm install --no-frozen-lockfile
+        run: pnpm install --frozen-lockfile
```

can be omitted https://pnpm.io/cli/install#--frozen-lockfile

```suggestion
        run: pnpm install
```

---

I like the extra security, I added them on purpose, they checks for CI environment, but I feel more robust by adding them 🦀

---

it's not an extra security it's how it works on CI environment

so it's secure by default on CI env :-)

---

Done 👍